### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by Incei**
…revious review.

### CODE REVIEW

**Comment by Incei** [comment]
@AndreDlm

File: Input/Qualification-Ontogeny-UGT1A9.json [hunk 1]
```diff
@@ -1,35 +1,30 @@
 {
-    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v2.1/OSP_Qualification_Plan_Schema.json",
+    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v3.0/OSP_Qualification_Plan_Schema.json",
     "Projects": [
         {
             "Id": "Propofol",
             "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Propofol-Model/master/Propofol.json",
```
Comment (Yuri05) [inline Input/Qualification-Ontogeny-UGT1A9.json:6]: PAth must be replace with the reference to the released version (once Propofol is released).
Will create an issue for that.

File: Input/Qualification-Ontogeny-UGT1A9.json [hunk 2]
```diff
@@ -1,35 +1,30 @@
...
                 {
                     "Type": "Compound",
                     "Name": "Propofol",
                     "Project": "Propofol"
                 }
             ]
+        },
+        {
+            "Id": "Propofol-Pediatrics",
+            "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Propofol-Pediatrics/main/Propofol-Pediatrics.json",
```
Comment (Yuri05) [inline Input/Qualification-Ontogeny-UGT1A9.json:17]: same

File: Input/Qualification-Ontogeny-UGT1A9.json [hunk 3]
```diff
@@ -1,35 +1,30 @@
...
+            "BuildingBlocks": []
         }
     ],
     "ObservedDataSets": [
         {
             "Id": "PK-Parameter",
-            "Path": "https://raw.githubusercontent.com/Incei/Database-for-observed-data/master/Pediatrics.csv",
+            "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Database-for-observed-data/master/Pediatrics.csv",
```
Comment (Yuri05) [inline Input/Qualification-Ontogeny-UGT1A9.json:24]: same

**Review by Yuri05 — CHANGES_REQUESTED** [review]
Workflow.R is not saved as UTF-8 and makes problems.
Please save as UTF-8

**Comment by Yuri05** [comment]
should it look like this?
grafik

**Comment by Yuri05** [comment]
please use either a picture for GMFE (https://github.com/Open-Systems-Pharmacology/Qualification-DDI-UGT/blob/master/Qualification/Input/Content/images/GFME_equation.PNG)

Or you could even use Latex 😉 
`$$ {\Huge  GMFE = 10^{\frac{\sum(|log(\frac{Pred}{Obs})|)}{n}} }  $$`

$$ {\Huge  GMFE = 10^{\frac{\sum(|log(\frac{Pred}{Obs})|)}{n}} }  $$

File: Input/Qualification-Ontogeny-UGT1A9.json [hunk 4]
```diff
@@ -144,381 +139,122 @@
...
-                                "ObservedData": "PK-Parameter",
-                                "ObservedDataRecordId": 5434
-                            }
-                        ]
-                    }
-                ]
-            },
             {
                 "Title": "Overall predictivity of the propofol PBPK model. Open circles represent mean ratios of PBPK predicted AUC over observed AUC of propofol in children 8 days to 10 years old. Blue dashed lines and red dotted lines represent the 1.5-fold and 2-fold error, respectively.",
                 "PKParameter": "AUC",
```
Comment (Yuri05) [inline Input/Qualification-Ontogeny-UGT1A9.json]: This is still accepted, but to make it conform with the new schema, it should be
`"PKParameters": ["AUC"],`

**Comment by Incei** [comment]
@Yuri05  : Please save as UTF-8 -->so system default encoding in R-studio is ISO-88-59-1.... meaning all my other workflows are also not UTF-8....
Not sure if others adjusted that either

**Comment by Yuri05** [comment]
>Please save as UTF-8 -->so system default encoding in R-studio is ISO-88-59-1....

Please change it to UTF-8 😉 
grafik

>meaning all my other workflows are also not UTF-8.... 

As long as they don't use non-ASCII characters (like "µ" etc.) - it's not a problem.

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**